### PR TITLE
Spelling: Chosen, and various simplifications

### LINKED ISSUE #2079: Choose/Select consistency in strings
This has been brought up here: https://hosted.weblate.org/translate/weblate/master/fr/?checksum=3205fc07a8b7d090#comments

> We have 14 occurrences of Choose and 5 Select. IMO, to ensure consistency, we should use the same term for the same concept.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/59369318-choose-select-consistency-in-strings?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by comradekingu**
Fixes #2079
Also, https://github.com/WeblateOrg/website/pull/33

### CODE REVIEW

File: weblate/auth/models.py [hunk 1]
```diff
@@ -114,12 +114,12 @@ class Group(models.Model):
     )
 
     project_selection = models.IntegerField(
-        verbose_name=_('Project selection'),
```
Comment (nijel) [inline weblate/auth/models.py]: This is confusing - this field determines how project selection (which is just next field) works.

File: weblate/auth/models.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ class Group(models.Model):
     )
 
     language_selection = models.IntegerField(
-        verbose_name=_('Language selection'),
```
Comment (nijel) [inline weblate/auth/models.py]: This is confusing - this field determines how langauge selection (which is just next field) works.

File: weblate/auth/models.py (hunk 1 above)
Comment (comradekingu) [reply weblate/auth/models.py]: Could it be "Choose language"?

Comment (nijel) [reply weblate/auth/models.py]: IMHO no, take a look at admin interface screenshot:

screenshot-2018-6-17 change group weblate administration

**Comment by nijel** [comment]
Thanks, most of the changes have been merged as 32d7a59beae87eba620182fcfb4a16157cc6e8ad. I've left out Project/Language selection part, see above discussion.

**Pull request closed without merging**